frontend/packages/console-dynamic-plugin-sdk/src/schema/console-extensions.ts

```diff
@@ -129,3 +129,7 @@ export type SupportedExtension =
  * Schema of Console plugin's `console-extensions.json` file.
  */
 export type ConsoleExtensionsJSON = SupportedExtension[];
+
+const CONSOLE_EXTENSION_PREFIX = 'console.';
+export const isConsoleExtension = (extension: SupportedExtension): boolean =>
```

Comment:
Type `SupportedExtension` here is not quite accurate sine it could be another, unknown type.